src/lightning/pytorch/utilities/model_summary/model_summary.py

```diff
@@ -50,7 +49,7 @@ class LayerSummary:
         >>> model = torch.nn.Conv2d(3, 8, 3)
         >>> summary = LayerSummary(model)
         >>> summary.num_parameters
-        224
+        tensor(224.)
```

Comment:
expected output for doctests

Reply:
So I should ensure that the output is `224` and not `tensor(224.)` ?

Reply:
I have updated the doctest for this as well